core/app/models/spree/variant/pricing_options.rb

```diff
@@ -1,30 +1,95 @@
 module Spree
   class Variant
+    # Instances of this class represent the set of circumstances that influence how expensive a
+    # variant is. For this particular pricing options class, country_iso and currency influence
+    # the price of a variant.
+    #
+    # Pricing options can be instantiated from a line item or from the view context:
+    # @see Spree::LineItem#pricing_options
+    # @see Spree::Core::ControllerHelpers::Pricing#current_pricing_options
+    #
     class PricingOptions
+      # When editing variants in the admin, this is the standard price the admin interacts with:
+      # The price in the admin's globally configured currency, for the admin's globally configured
+      # country. These options get merged with any options the user provides when instantiating
+      # new pricing options.
+      # @see Spree::Config.default_pricing_options
+      # @see #initialize
+      # @return [Hash] The attributes that admin prices usually have
+      #
       def self.default_price_attributes
-        { currency: Spree::Config.currency }
+        {
+          currency: Spree::Config.currency,
+          country_iso: Spree::Config.admin_vat_country_iso
```

Comment:
why the `admin_vat_country_iso` is used here, rather than `default_country_iso`?